api/src/service/domain/workflow/subproject_projected_budget_delete.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { assert } from "chai";
```

Comment:
Please add the following tests:

- Delete existing PB and check if it is indeed deleted
- Delete non-exsiting PB and check that no error occurs

Reply:
Done
also added unit test for updating PBs (project and subproject)